### PR TITLE
[Expert] Guns without roses

### PR DESCRIPTION
**Pull request by MuteTiefling**
Iron
image

Gold
image

Diamond Shotgun
image

Diamond Sniper
image

Diamond Gatling
image

Phantasmal Rifle
image

Jagershot
image

### CODE REVIEW

**Comment by MuteTiefling** [comment]
Hunger Buillet
image

**Comment by MuteTiefling** [comment]
Blaze Bullets
image

Iron Bullets, made cheaper. more in line with IE ammo
image

Same for flint bullets. 
image

File: kubejs/server_scripts/enigmatica/kubejs/expert/recipetypes/thermal/machine/casting.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+onEvent('recipes', (event) => {
+    if (global.isExpertMode == false) {
```
Comment (NillerMedDild) [inline kubejs/server_scripts/enigmatica/kubejs/expert/recipetypes/thermal/machine/casting.js:2]: Filename is incorrect, and there's no id prefix or recipe name :)

Comment (MuteTiefling) [reply kubejs/server_scripts/enigmatica/kubejs/expert/recipetypes/thermal/machine/casting.js]: Oh, worse than that, this file really was useless. The recipe is already in `induction_smelter` where it belongs.

**Comment by NillerMedDild** [comment]
Nice recipes! :D

**Pull request merged**